doc/audit/builtin_maps.rst

```diff
@@ -143,14 +143,14 @@ Versions of the code allowed to join the current network on :doc:`SGX <../operat
 
 .. note:: On versions ``3.0.0-dev1`` and below, value was an enum with one possible value ``ALLOWED_TO_JOIN``
 
-``nodes.security_policies``
+``nodes.host_data``
```

Comment:
nodes.snp.host_data, or nodes.snp_host_data (I would prefer the former, just to group the snp tables)

Reply:
5143a02dcb85d2aacabb5c1fd88e756aed95166a